Godeps

```diff
@@ -73,8 +73,8 @@
 			"Rev": "43bd4e6d53ee382fd09bb6ede8b4029247444945"
 		},
 		{
-			"ImportPath": "github.com/samuel/go-zookeeper/zk",
-			"Rev": "4a222ca51b4c7ca6f2895e0e68af2e9254acfdcc"
+			"ImportPath": "github.com/zenoss/go-zookeeper/zk",
+			"Rev": "c34ab6262b97ef46e1ab13dd87cfca6de2042a15"
 		},
```

Comment:
Unfortunately, no. There are a couple negative values that have meaning (-2 : ping)